tsconfig.json

```diff
@@ -19,7 +19,11 @@
     "noFallthroughCasesInSwitch": true,
     "resolveJsonModule": true,
     "moduleResolution": "node",
-    "typeRoots": ["node_modules/@types", "src/types/*"],
+    "baseUrl": "./",
+    "paths": {
+      "*": ["src/types/*"]
+    },
+    "typeRoots": ["node_modules/@types"],
```

Comment:
Why the change of heart? Seems more confusing

Reply:
npm could not find my types with typeRoots